webrender/src/platform/unix/font.rs

```diff
@@ -159,74 +159,71 @@ impl FontContext {
                                                               glyph_height as usize *
                                                               4);
 
-                    match bitmap_mode {
-                        FT_PIXEL_MODE_MONO => {
-                            // This is not exactly efficient... but it's only used by the
-                            // reftest pass when we have AA disabled on glyphs.
-                            let offset_x = (metrics.horiBearingX >> 6) as i32 - (*slot).bitmap_left;
-                            let offset_y = (metrics.horiBearingY >> 6) as i32 - (*slot).bitmap_top;
-
-                            // Due to AA being disabled, the bitmap produced for mono
-                            // glyphs is often smaller than the reported glyph dimensions.
-                            // To account for this, place the rendered glyph within the
-                            // box of the glyph dimensions, filling in invalid pixels with
-                            // zero alpha.
-                            for iy in 0..glyph_height {
-                                let y = iy - offset_y;
-                                for ix in 0..glyph_width {
-                                    let x = ix + offset_x;
-                                    let valid_byte = x >= 0 &&
-                                                     y >= 0 &&
-                                                     x < bitmap.width &&
-                                                     y < bitmap.rows;
-                                    let byte_value = if valid_byte {
-                                        let byte_index = (y * bitmap.pitch) + (x >> 3);
-                                        let bit_index = x & 7;
-                                        let byte_ptr = bitmap.buffer.offset(byte_index as isize);
-                                        let bit = (*byte_ptr & (0x80 >> bit_index)) != 0;
-                                        if bit {
-                                            0xff
-                                        } else {
-                                            0
-                                        }
+                    if bitmap.pixel_mode == FT_Pixel_Mode::FT_PIXEL_MODE_MONO as u8 {
```

Comment:
Because now `FT_PIXEL_MODE_XXX` is a proper enum, and to match you'd need to transmute, or write a function to convert u8 -> FT_Pixel_Mode, but I thought it was not worth it.